src/app/QuickStarts/quickstarts/settings-quickstart.tsx

```diff
@@ -54,98 +54,95 @@ const SettingsQuickStart: QuickStart = {
     displayName: 'Using Settings',
     durationMinutes: 5,
     icon: <CogIcon />,
-    description: `Learn about the settings page in [APP] and how to use it.`,
+    description: `Learn about the settings page in **[APP]** and how to use it.`,
     prerequisites: [''],
     introduction: `
-  <div style="padding-bottom: 1rem">
-    <p>
-    <h1>Using Settings</h1>
-      Cryostat has a settings page that allows you to configure the application. This quick start will show you how to use the settings page.
-      <h3>
-      There are various settings that can be configured:
-      </h3>
-      <ul style="font-weight: bold;">
-        <li>General</li>
-        <li>Connectivity</li>
-        <li>Notification & Messages</li>    
-        <li>Dashboard</li>
-        <li>Advanced</li>
-      </ul>
-      We will go over each of these settings briefly.
-    </p>
-  </div>
+## Using Settings
+      
+[APP] provides a settings page that lets you configure various aspects of the application. This quick start will show you how to use the settings page.
+
+There are various settings that can be configured:
+* General
+* Connectivity
+* Notification & Messages   
+* Dashboard
+* Advanced
+
+### What you'll learn
+  - How to navigate to the settings page
+  - How to configure settings in [APP]
     `,
     tasks: [
       {
         title: 'Navigate to the Settings page',
         description: `
-1. Press the [Settings]{{highlight settings-link}} cog icon.`,
+1. On the masthead, click the [Settings]{{highlight settings-link}} cog.`,
       },
       {
         title: 'Go to the General settings tab',
         description: `
-1. Go to the [General]{{highlight settings-general-tab}} tab
+1. Click [General]{{highlight settings-general-tab}}.
```

Comment:
Lets include a short intro to what general settings do.

Maybe like `configure app appearances such as themes and lanaguages, ...`